src/Arcus.Scripting.PS.KeyVault/Scripts/Get-Infra-KeyVaultAccessPolicies.ps1

```diff
@@ -0,0 +1,57 @@
+param(
```

Comment:
I would rename this script to `Get-KeyVaultAccessPolicies` to be honest, `Infra` is legacy from our internal system if you ask me.

Reply:
Or should we align and use `Get-AzKeyVaultAccessPolicies` similar to the official cmdlets? Maybe not a bad thing?

Reply:
Renamed this.